{ 
  "text" : "fs/f2fs/extent_cache.c in the Linux kernel before 4.13 mishandles extent trees, which allows local users to cause a denial of service (BUG) via an application with multiple threads.\n" ,
  "entities" : [ { 
    "eid" : "E1" ,
    "keyid" : "FilePath#c84b1c44d69dd1d5bd7513a73cce843e" ,
    "type" : "FilePath" ,
    "subType" : "FilePath" ,
    "value" : "/f2fs/extent_cache" ,
    "begin" : 2 ,
    "end" : 20 ,
    "canonicalForm" : "/f2fs/extent_cache" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E2" ,
    "keyid" : "Identity#e206a54e97690cce50cc872dd70ee896" ,
    "type" : "Identity" ,
    "subType" : "Organization" ,
    "value" : "linux" ,
    "begin" : 30 ,
    "end" : 35 ,
    "canonicalForm" : "Linux" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E3" ,
    "keyid" : "Product#e206a54e97690cce50cc872dd70ee896" ,
    "type" : "Product" ,
    "subType" : "OperatingSystem" ,
    "value" : "linux" ,
    "begin" : 30 ,
    "end" : 35 ,
    "canonicalForm" : "Linux" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E4" ,
    "keyid" : "TTP#9c98ef6a7451d6f82ca2194ee23d1acb" ,
    "type" : "TTP" ,
    "subType" : "TTP" ,
    "value" : "denial of service" ,
    "begin" : 116 ,
    "end" : 133 ,
    "canonicalForm" : "denial of service" ,
    "score" : "0.0"
  }
  ,  ] ,
  "relations" : [  ]
}
